intlist.c

```diff
@@ -0,0 +1,269 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// intlist.c
+// Program on main branch.
+// ------------------------------------------------------------------------------------------------
+
+
+#include <stdio.h>
+#include <malloc.h>
+#include "intlist.h"
+
+
+struct node {
+   int value;
+   struct node* link;
+};
+
+//Defines head pointer and fixes it to null
+struct node* Create () {
+   struct node* head;

```

Comment:
intlist.c and intlist.h are library files. They should not print any messages. If communication is required, you can return an integer and have a corresponding error code (for eg: return -1 for empty list and so on). You can mention this in the summary tag comment